bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -1237,6 +1237,43 @@ private CaseDbSchemaVersionNumber updateFromSchema7dot1toSchema7dot2(CaseDbSchem
 		}
 	}
 
+	/**
+	 * Updates a schema version 7.2 database to a schema version 7.3 database.
+	 *
+	 * @param schemaVersion The current schema version of the database.
+	 * @param connection    A connection to the case database.
+	 *
+	 * @return The new database schema version.
+	 *
+	 * @throws SQLException     If there is an error completing a database
+	 *                          operation.
+	 * @throws TskCoreException If there is an error completing a database
+	 *                          operation via another SleuthkitCase method.
+	 */
+	private CaseDbSchemaVersionNumber updateFromSchema7dot2toSchema7dot3(CaseDbSchemaVersionNumber schemaVersion, CaseDbConnection connection) throws SQLException, TskCoreException {
+		if (schemaVersion.getMajor() != 7) {
+			return schemaVersion;
+		}
+
+		if (schemaVersion.getMinor() != 2) {
+			return schemaVersion;
+		}
+
+		Statement statement = null;
+		Statement updstatement = null;
```

Comment:
Sort of an unusual abbreviation, is it intentional or a typo?

Reply:
No idea...copy and paste from previous update code. I'll take care of it when I get to filling out this method.

Reply:
Okay, thanks.